ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -85,14 +75,14 @@ def _format_single_funnel(self, result, with_breakdown=False):
 
             if result and len(result) > 0:
                 total_people += result[step.order]
-                relevant_people += result[cast(int, step.order) + num_entities]
 
-            serialized_result = self._serialize_step(step, total_people, relevant_people[0:100])
+            serialized_result = self._serialize_step(step, total_people, [])
             if cast(int, step.order) > 0:
+
                 serialized_result.update(
                     {
-                        "average_conversion_time": result[cast(int, step.order) + num_entities * 2 - 1],
-                        "median_conversion_time": result[cast(int, step.order) + num_entities * 3 - 2],
+                        "average_conversion_time": result[cast(int, step.order) + num_entities * 1 - 1],
```

Comment:
What are these changes for?

Reply:
We removed `n` rows from the result (people arrays), so the placement of others in the result changes